aten/src/ATen/native/native_functions.yaml

```diff
@@ -3418,6 +3418,15 @@
   variants: function, method
   supports_named_tensor: True
 
+- func: is_lazy(Tensor self) -> bool
+  variants: method
+
+- func: to_lazy(Tensor self) -> Tensor
+  variants: method
+
+- func: to_eager(Tensor self) -> Tensor
+  variants: method
```

Comment:
Could we underscore these methods to make them private for now?

Reply:
a trailing underscore? otherwise:

```
RuntimeError: Function '_to_lazy' starts with a single underscore and is configured to have a method on Tensor. Functions that start with  a single underscore should only be functions in the at:: namespace and not methods on Tensor!
```

Reply:
Lol, I guess it would have to be a function hahaha

Reply:
(Please not a trailing underscore, that means inplace!)

Reply:
To clarify, yes, let's make them functions, in that case.